crates/fuel-streams-core/src/types.rs

```diff
@@ -66,6 +80,21 @@ pub enum TransactionStatus {
     Success,
 }
 
+impl From<FuelTransactionStatus> for TransactionStatus {
```

Comment:
This is also already [implemented here](https://github.com/FuelLabs/data-systems/blob/main/crates/fuel-streams-core/src/transactions/types.rs)

Reply:
used